apps/prairielearn/src/pages/instructorAssessments/instructorAssessments.sql

```diff
@@ -15,28 +15,7 @@ WITH
       a.id
   )
 SELECT
-  a.id,
-  a.tid,
-  a.course_instance_id,
-  a.type,
-  a.number as assessment_number,
-  a.title,
-  a.group_work AS group_work,
-  a.assessment_set_id,
-  a.sync_errors,
-  a.sync_warnings,
-  a.score_stat_number,
-  a.score_stat_mean,
-  a.score_stat_std,
-  a.score_stat_min,
-  a.score_stat_max,
-  a.score_stat_median,
-  a.score_stat_n_zero,
-  a.score_stat_n_hundred,
-  a.score_stat_n_zero_perc,
-  a.score_stat_n_hundred_perc,
-  a.score_stat_hist,
-  format_interval (a.duration_stat_mean) AS duration_stat_mean_formatted,
+  a.*,
```

Comment:
We have started using `to_json(a)` when passing back entire rows. Do we want to do that here?

Reply:
Since the assessment itself is the main "target" of the question I'd be fine either way, but I'd defer to @nwalters512 on if this should be the case here.

Reply:
Because this is trying to replicate the existing behaviour as much as possible I'll keep this as is for now, we can follow up in the future to change this if needed.